taxi_service/settings.py

```diff
@@ -108,6 +108,7 @@
 
 AUTH_USER_MODEL = "taxi.Driver"
```

Comment:
Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension) as per the checklist requirement.